modules/base/dashboard/dashboarditemdistance.cpp

```diff
@@ -440,18 +445,20 @@ void DashboardItemDistance::render(glm::vec2& penPosition) {
     }
 
     penPosition.y -= _font->height();
-
-    RenderFont(
-        *_font,
-        penPosition,
-        fmt::format(
-            "Distance from {} to {}: {:f} {}",
-            sourceInfo.second, destinationInfo.second, dist.first, dist.second
-        )
+    std::fill(_buffer.begin(), _buffer.end(), 0);
```

Comment:
It would be good to have a limitation in place to keep the length of a scene graph node name/identifier to < 100 characters to be able to have a reasonable limit on the length of the formatted string